test/functional/provider/nodejs_spec.lua

```diff
@@ -0,0 +1,70 @@
+local helpers = require('test.functional.helpers')(after_each)
+local eq, clear = helpers.eq, helpers.clear
+local missing_provider = helpers.missing_provider
+local command = helpers.command
+local write_file = helpers.write_file
+local eval = helpers.eval
+local sleep = helpers.sleep
+local funcs = helpers.funcs
+local retry = helpers.retry
```

Comment:
After doing:

    sudo npm install -g neovim

these tests fail on my system. I think `NODE_PATH` should be set in the tests, not in CI scripts only.